ui.apps/src/main/content/jcr_root/apps/testaem/components/helloworld/helloworld.html

```diff
@@ -13,8 +13,8 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
-<div class="cmp-helloworld" data-cmp-is="helloworld">
-    <h2 class="cmp-helloworld__title">Hello World Component !</h2>
+<img class="cmp-helloworld" data-cmp-is="helloworld">
```

Comment:
why is img tag inside another img? :)

Reply:
Madness? No this is "HELLO WORLD"

corrected in the last commit.